packages/ERTP/src/displayInfo.js

```diff
@@ -67,6 +70,19 @@ export const coerceDisplayInfo = (allegedDisplayInfo, assetKind) => {
   assertSubset(displayInfoKeys, Object.keys(displayInfo));
   if (displayInfo.decimalPlaces !== undefined) {
     assert.typeof(displayInfo.decimalPlaces, 'number');
+    assert(
+      Number.isSafeInteger(displayInfo.decimalPlaces),
+      X`decimalPlaces ${displayInfo.decimalPlaces} is not a safe integer`,
+    );
```

Comment:
I think the second assert subsumes the first, so the first can be deleted.
```suggestion
    assert(
      Number.isSafeInteger(displayInfo.decimalPlaces),
      X`decimalPlaces ${displayInfo.decimalPlaces} is not a safe integer`,
    );
```